### PR TITLE
Feature filter on chr 200603

### PR DESCRIPTION
**Pull request by mikaell**
This PR adds functionality to filter cancer variants on chromosome, start/end in cancer view.

Ref #1926 -customer request.

The functionality is like filtering on the ordinary SNVs/Indels and uses the same backend.

**How to test**:
1. how to test it, possibly with real cases/data
Install `featuer-filter_on_chr-200603` (typo!).

Go to cancer variants, select filter and enter a chromosome, start and end. 

**Expected outcome**:
All matching variants will be shown.

Screenshot 2020-06-03 at 14 44 55

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
> 1. Install `featuer-filter_on_chr-200603` (typo!).

😆

**Comment by northwestwitch** [comment]
@mikaell write an explicit comment when you are done with this PR, so we know and someone reviews!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Show an ellipsis if 10 cases or more to display with loqusdb matches
 - A new blog post for version 4.17
 - Tooltip to better describe Tumor and Normal columns in cancer variants
+- Filter on chromosome start/end in Cancer View
```
Comment (northwestwitch) [inline CHANGELOG.md:14]:
```suggestion
- Filter cancer SNVs and SVs variants by chromosome coordinates
```

File: scout/server/blueprints/variants/forms.py [hunk 2]
```diff
@@ -105,6 +105,10 @@ class VariantFiltersForm(FlaskForm):
     load_filter = SubmitField(label="Load filter")
     delete_filter = SubmitField(label="Delete filter")
 
+    chrom = TextField("Chromosome", [validators.Optional()])
```
[the remark this answers is not in the document]
Comment (mikaell) [reply scout/server/blueprints/variants/forms.py]: 1. Changed to select-tag.

2. Refactored code 
(Those changes were made but seem to have disappeared)

**Comment by mikaell** [comment]
> The new form fields work but I have a few suggestions to improve it, see my comments.
> 
> We have recently introduced an additional search by cytoband in the other types of variants filter forms pages. I didn't add it in cancer pages because I knew that you were working on this code at the moment and didn't want to make a hell of conflicts.
> 
> But now It would be good to include the search by cytoband also for cancer variants (both cancer SNVs and SVs). But it doesn't have to be done in this PR, could be another one as well. It's up to you really.

Better to do it in a it's own PR.

**Comment by mikaell** [comment]
> The new form fields work but I have a few suggestions to improve it, see my comments.
> 
> We have recently introduced an additional search by cytoband in the other types of variants filter forms pages. I didn't add it in cancer pages because I knew that you were working on this code at the moment and didn't want to make a hell of conflicts.
> 
> But now It would be good to include the search by cytoband also for cancer variants (both cancer SNVs and SVs). But it doesn't have to be done in this PR, could be another one as well. It's up to you really.

Additional cytoband searches will be a separate PR.

_Now I'm just trying to close all issues to allow merging, this has already been discussed_

File: scout/server/blueprints/variants/forms.py [hunk 3]
```diff
@@ -117,8 +121,6 @@ class FiltersForm(VariantFiltersForm):
     chrom = SelectField("Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS)
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/forms.py:121]: This one can also go, because it will be inherited from VariantFiltersForm

**Pull request merged**